pkgs/development/python-modules/sharp_aquos_rc/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "sharp_aquos_rc";
+  version = "0.4";
```

```suggestion
  version = "0.4";
  pyproject = true;
```

Use the pyproject format and set `nativeBuildInputs` explicitly.